### DISCUSSION TITLE
executor in aiogram v3

### DISCUSSION THREAD

**Discussion by KVInventoR** [aiogram/aiogram#1171] [General]
Is there any example, how to get using executor in aigram v3?
I see that, executor was present in v2, but I didn't find anything in v3

For me, it will be good, if current example folder also contains some examples with using aiogram and parallel loops.

**Comment by affkie** [comment]
Executor is the Aiogram 2.x feature, not 3.x.

**Reply by KVInventoR** [reply]
Is there any example, which can help to replace logic with executor?
I have 2 parallel loops, which running, during starting bot, they are watching local folders and sending messages to users.

**Comment by JrooTJunior** [comment]
Executor is no more needed. Just look at the documentation or examples directory, you can start polling from dispatcher or webhook via aiohttp or with any other web-framework that supports asyncio mode.

If you have any troubles with examples specified in the documentation or repo feel free to provide code examples and errors that you have and the we can help you.

**Reply by iQiexie** [reply]
how do i run my custom polling? Can you give link to that documentation part, please?

**Reply by Olegt0rr** [reply]
What is custom polling? :)

**Comment by sxcntqnt** [comment]
hello did you get passs this issue

**Reply by JrooTJunior** [reply]
Just read migration guide: https://docs.aiogram.dev/en/dev-3.x/migration_2_to_3.html

**Reply by DAYAVKURSE** [reply]
it is never fully described

**Reply by JrooTJunior** [reply]
What actually you don't understand?

**Comment by QRage** [comment]
`The Executor has been entirely removed; you can now use the Dispatcher directly to start poll the API or handle webhooks from it.`
—Migration Docs.aiogram

Just run it with Dispatcher. There shall be no problem.
